opcua.cpp

```diff
@@ -1040,6 +1042,11 @@ void OPCUA::setTraceFile(const std::string &traceFile)
 {
 	if (traceFile == "True" || traceFile == "true" || traceFile == "TRUE")
```

Comment:
@[ashwini-k-pandey](https://github.com/ashwini-k-pandey) ashwini-k-pandey [now](https://github.com/fledge-iot/fledge-south-s2opcua/pull/74/files#r1868131775)
FYI.. I am adding static bool stringToBool(const std::string& str) in PR https://github.com/fledge-iot/fledge/pull/1504 .